bindings/netstandard/ElectionGuard/ElectionGuard.ElectionSetup/KeyCeremonyMediator.cs

```diff
@@ -213,16 +213,20 @@ private IEnumerable<string> GetAnnouncedGuardians()
     /// </summary>
     /// <param name="guardianId">Recipients guardian id</param>
     /// <returns>List of guardians designated backups</returns>
-    private List<ElectionPartialKeyBackup> ShareElectionPartialKeyBackupsToGuardian(string guardianId)
+    private List<ElectionPartialKeyBackup> ShareElectionPartialKeyBackupsToGuardian(
+        string guardianId)
     {
         List<ElectionPartialKeyBackup> backups = new();
         var announcedGuardians = GetAnnouncedGuardians();
-        var others = announcedGuardians.Where(g => g != guardianId);
+        var others = announcedGuardians;

```

Comment:
I believe this is correct, it is according to the spec and the passing tests, but it does seem a bit superfluous to share our own backups to ourselves, nevertheless it is consistent with the current behavior of hydration from the database. Im not sure if removing this where clause will have an effect on the key ceremony validation in the UI or not